modular_skyrat/modules/loadouts/loadout_ui/loadout_outfit_helpers.dm

```diff
@@ -30,6 +30,13 @@
 	else
 		CRASH("Outfit passed to equip_outfit_and_loadout was neither a path nor an instantiated type!")
 
+	if(outfit_override)
+		for(var/outfit_var in outfit_override)
+			if(!ispath(outfit_override[outfit_var]) && !isnull(outfit_override[outfit_var]))
+				CRASH("outfit_override var on [real_name] spawner has incorrect values! it must be an assoc list with outfit \"var\" = path | null")
+			outfit.vars[outfit_var] = outfit_override[outfit_var]
```

Comment:
What's the purpose of this?

Also directly accessing vars isn't great in the slightest.

Reply:
It's to support whatever admin overrides are set for custom spawners, or at least that's what the TG comments say.

Reply:
We've never really had issues with it before from what I know, so I'm fine with reverting that change if you want me to.

Reply:
> or at least that's what the TG comments say.

Where did it say that?